content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Installation (Installer Beta)"
+sidebar_current: "docs-enterprise2-private-install-installer"
+---
+
+# Private Terraform Enterprise Automated Installation (Installer Beta)
+
+The installation of Private Terraform Enterprise can be completely automated, for both online and airgapped modes.  Before invoking the installer you must:
+- prepare `/etc/replicated.conf`
+- prepare an [application settings](#application-settings) file
+- copy your license file to the instance
+- download the `.airgap` bundle to the instance (Airgapped mode only)
+
+You may also need to provide additional flags (such as the instance's public and private IP addresses) in order to not be prompted for those values when running the installer.
+
+It's expected that the user is already familiar with how to do a [manual install](./install-installer.html#installation).
+
+## Application settings
```

Comment:
thoughts on having the `replicated.conf` section first since it happens first in the install process?

Reply:
The application settings are referenced _in_ `replicated.conf`, and they're also the same across install modes.  I was trying to reduce duplication, but clarity is more important.